### PR TITLE
Feature/county display edits

### LINKED ISSUE #1148: Display of unlisted counties in App Detail looks unweildy
screen shot 2017-09-11 at 1 55 52 pm

### PR DESCRIPTION
**Pull request by bengolder**
Closes #1148 

### What does this do and why?

This changes the language about where an applicant needs help on both the application detail and application printout pages.

### Testing & Checks

What does this PR include?
- [ ] new dependencies
- [ ] migrations
- [ ] data migrations
- [ ] celery tasks
- [ ] changes to environment variables or local settings
- [ ] configuration changes for 3rd party services (Travis, Heroku, AWS, Github, etc) 
- [x] visual or style changes

Are there any human checks that should be used to verify these changes, such as user acceptance tests (UAT) or manual deployment checks?

We should make sure that the no listed application process and the resulting app detail and printout both appear satisfactory.

Any integration or unit tests?
There are unit tests for customizations to the Counties field.

### CODE REVIEW

File: formation/templates/formation/intake_display.jinja [hunk 1]
```diff
@@ -78,7 +80,7 @@
       {{ form.serving_sentence_display }}
     </div>
     <div class="data_display-stack">
-      {{ form.has_been_to_prison }}
+      {{ form.has_been_to_prison_display }}
```
Comment (bengolder) [inline formation/templates/formation/intake_display.jinja:83]: This was an unrelated bug that I noticed while UATing this fix

File: formation/fields.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 from django.utils.safestring import mark_safe
 from django.utils.translation import ugettext_lazy as _
 from django.core.validators import EmailValidator, URLValidator
+from project.jinja2 import oxford_comma
+from markupsafe import escape
```
Comment (rossettistone) [inline formation/fields.py]: I don't think this is used anywhere in this file. Maybe a leftover?

Comment (bengolder) [reply formation/fields.py]: escape is, but oxford comma is used

**Review by rossettistone — APPROVED** [review]
Looks good!

**Pull request merged**